pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -255,7 +259,15 @@ async def _update_nightly_imagestreams(self):
             cmd.extend([f'--exclude-arch {arch}' for arch in self.exclude_arches])
         if self.runtime.dry_run:
             cmd.extend(['--skip-gc-tagging', '--moist-run'])
-        await exectools.cmd_assert_async(cmd)
+
+        rc, stdout, stderr = await exectools.cmd_gather_async(cmd)
+        viable_false_message = "DO NOT PROCEED WITH THIS ASSEMBLY PAYLOAD"
+        if rc != 0:
```

Comment:
If `rc != 0` `cmd_gather_async` will raise a `ChildProcessError` since `check` is True by default. We don't need to check it